pallets/roles/src/tests.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of Webb.
```

Comment:
Change for all files.
```suggestion
// This file is part of Tangle.
```